spec/controllers/documents_controller_spec.rb

```diff
@@ -189,6 +189,104 @@
       end
     end
 
+    context 'enrolled' do
+      let(:application) do
```

Comment:
I think loading applicant and application is not required for verification types. Can you remove this?